src/components/Dialog/DialogFooter.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { Modal } from "../Modal"
+
+export const DialogFooter = ({ className, children,...rest }: React.ComponentProps<typeof Modal.Footer>) => {
```

Comment:
```suggestion
export const DialogFooter = ({ className, children,...rest }: Omit<React.ComponentProps<typeof Modal.Footer>, 'hideBorder>) => {
```
because we don't want to let the developers to controll the border.